CODE_OF_CONDUCT.md

```diff
@@ -1,21 +1,16 @@
-# Code of Conduct
+# The Creator's Code
 
-We are sentient.  
-To be sentient is to be limited.  
-In our limitation, we make choices that are unwise or are flawed.
+Because we are limited beings,
+we make choices that are unwise or flawed.
 
-If we make unwise choices because of our limitation,  
-we cannot judge others for the same reason.
+While others may meet us with hostility,
+we choose to acknowledge difference:
```

Comment:
Here comes the antistrophe - the reply and balance to the opening.

In your original you give us a sort of categorical imperative (i.e. a universal rule) framed both in terms of ourselves and others (I really like that aspect).

However, upon reflection, I'm not sure about "judge" used in a negative sense. I "judge" things all the time -- for example, I've judged this as a worthwhile, interesting and thoughtful project..! ;-) Also, just because I make unwise choices, does that mean I cannot still make judgements..? In fact, I'd argue that judgement is an essential aspect of making choices... if you have good judgement, you'll make good choices, if you see what I mean. Only by reflecting on one's bad judgement will you move towards good judgement - so in a sense, bad judgement is an essential part of growing and enlarging one's world view. (Does this make sense..?)

So, I've tried to retain the feeling of a universal declaration with reference to ourselves and others. I've also framed it as a conscious choice (to acknowledge difference in the face of prejudice, intolerance and so on) which leads us to the most important line of the text...